Sources/PostgreSQL/Connection.swift

```diff
@@ -171,6 +180,37 @@ public final class Connection: ConnInfoInitializable {
         }
     }
     
+    /// Creates a dispatch read source for this connection that will call `callback` on `queue` when a notification is received
+    ///
+    /// - Parameter channel: the channel to register for
+    /// - Parameter queue: the queue to create the DispatchSource on
+    /// - Parameter callback: the callback
+    /// - Parameter note: The notification received from the database
+    /// - Parameter err: Any error while reading the notification. If not nil, the source will have been canceled
+    /// - Returns: the dispatch socket to activate
+    /// - Throws: if fails to get the socket for the connection
+    public func makeListenDispatchSource(toChannel channel: String, queue: DispatchQueue, callback: @escaping (_ note: Notification?, _ err: Error?) -> Void) throws -> DispatchSourceRead {
```

Comment:
I would simply call this function `listen(toChannel:queue:callback:)`

Reply:
Will do. I'd just copied Apple's naming convention. Better to use a swifty one.